Patches.py

```diff
@@ -1743,40 +1743,44 @@ def update_scrub_text(message, text_replacement, default_price, price, item_name
         write_rom_item(rom, 0x5B, item)
 
     # Update chest type appearance
-    if world.settings.correct_chest_appearances == 'textures':
-        symbol = rom.sym('CHEST_TEXTURE_MATCH_CONTENTS')
-        rom.write_int32(symbol, 0x00000001)
-    if world.settings.correct_chest_appearances == 'sizes':
-        symbol = rom.sym('CHEST_SIZE_MATCH_CONTENTS')
-        rom.write_int32(symbol, 0x00000001)
-        # Move Ganon's Castle's Zelda's Lullaby Chest back so is reachable if large
-        if not world.dungeon_mq['Ganons Castle']:
-            chest_name = 'Ganons Castle Light Trial Lullaby Chest'
-            location = world.get_location(chest_name)
-            item = read_rom_item(rom, location.item.index)
-            if item['chest_type'] in (GOLD_CHEST, GILDED_CHEST, SKULL_CHEST_BIG):
-                rom.write_int16(0x321B176, 0xFC40) # original 0xFC48
-
-        # Move Spirit Temple Compass Chest if it is a small chest so it is reachable with hookshot
-        if not world.dungeon_mq['Spirit Temple']:
-            chest_name = 'Spirit Temple Compass Chest'
-            chest_address = 0x2B6B07C
-            location = world.get_location(chest_name)
-            item = read_rom_item(rom, location.item.index)
-            if item['chest_type'] in (BROWN_CHEST, SILVER_CHEST, SKULL_CHEST_SMALL):
-                rom.write_int16(chest_address + 2, 0x0190) # X pos
-                rom.write_int16(chest_address + 6, 0xFABC) # Z pos
-
-        # Move Silver Gauntlets chest if it is small so it is reachable from Spirit Hover Seam
-        if world.settings.logic_rules != 'glitchless':
-            chest_name = 'Spirit Temple Silver Gauntlets Chest'
-            chest_address_0 = 0x21A02D0  # Address in setup 0
-            chest_address_2 = 0x21A06E4  # Address in setup 2
-            location = world.get_location(chest_name)
-            item = read_rom_item(rom, location.item.index)
-            if item['chest_type'] in (BROWN_CHEST, SILVER_CHEST, SKULL_CHEST_SMALL):
-                rom.write_int16(chest_address_0 + 6, 0x0172)  # Z pos
-                rom.write_int16(chest_address_2 + 6, 0x0172)  # Z pos
+    if world.settings.correct_chest_appearances != 'off':
```

Comment:
No need for putting the CSMC/CTMC changes in the same if block as the red ice changes in my opinion. Also makes it easier to merge with other open PRs if separated.

Reply:
I removed the Python changes when changing it to also affect red ice without chests.